front/.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "vue-app"
+    "vue-app-bar"
```

Comment:
what its for? there no such preset as vue-app-bar

Reply:
Resolved.